modules/core/src/main/java/org/apache/synapse/mediators/builtin/LogMediator.java

```diff
@@ -288,6 +302,16 @@ public void setCategory(int category) {
         }
     }
 
+    public String getMessageTemplate() {
+
```

Comment:
Why an additional new line?

Reply:
It's added by the `wso2_code_style` for idea.